### PR TITLE
Fix PathMapper not handling . and ..

### PR DESCRIPTION
**Pull request by abrooksv**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Description
If you had a PyCharm project with a CodeUri of `.` we failed to add breakpoints

## Related Issue(s)
#908 

## Testing
New unit test

Before fix:
```
Connected to pydev debugger (build 191.6183.87)
pydev debugger: warning: trying to add breakpoint to file that does not exist: /IdeaProjects/untitled1/hello_world/app.py (will have no effect)
START RequestId: 52fdfc07-2182-154f-163f-5f0f9a621d72 Version: $LATEST
{"statusCode":200,"body":"{\"message\": \"hello world\"}"}
END RequestId: 52fdfc07-2182-154f-163f-5f0f9a621d72
REPORT RequestId: 52fdfc07-2182-154f-163f-5f0f9a621d72	Init Duration: 1591.32 ms	Duration: 8.70 ms	Billed Duration: 100 ms	Memory Size: 128 MB	Max Memory Used: 33 MB	
```

After:
Breakpoint is hit

## Checklist

- [ ] I have read the **README** document
- [ ] I have read the **CONTRIBUTING** document
- [ ] Local run of `gradlew check` succeeds
- [ ] My code follows the code style of this project
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [ ] A short description of the change has been added to the **CHANGELOG**

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by Rubyj** [comment]
Looks nice! @abrooksv

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/PathMapper.kt [hunk 1]
```diff
@@ -52,13 +52,13 @@ class PathMapper(private val mappings: List<PathMapping>) {
                 localPath
             }
 
-            return FileUtil.normalize(updatedPath)
+            return FileUtil.toCanonicalPath(updatedPath)
```
Comment (zhangzhx) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/PathMapper.kt]: Not sure how likely customers will have symlinks in the path. Do you think it is safer to use 
`toCanonicalPath(updatedPath, true)` instead as described [here](https://github.com/JetBrains/intellij-community/blob/master/platform/util/src/com/intellij/openapi/util/io/FileUtil.java#L633)?

Comment (abrooksv) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/PathMapper.kt]: Couldn't hurt, also the integ tests are broken, so fixing them

**Comment by abrooksv** [comment]
Job aws-jetbrains-toolkit-integ-test:d49f5be3-e3f5-40d6-b278-4443712472d7 passed

**Pull request merged**